beta-src/src/models/BoardClass.ts

```diff
@@ -0,0 +1,83 @@
+import { IBoard } from "./Interfaces";
+import TerritoryClass from "./TerritoryClass";
+import UnitClass from "./UnitClass";
+
+// this is will be used for accessing all necessary data and methods
```

Comment:
nit: in general, replace "will be" with "is". here you have `is will be` lol

```
// this is used for accessing all necessary data and methods
```

isn't this true for all classes that have data and methods? let's make the comment more descriptive to someone who doesn't know what this class is doing. let's use this comment style:

```
/**
 * this is the comment style we want
 * /
```
```